integration-tests/bats/sql-server.bats

```diff
@@ -764,4 +764,24 @@ SQL
     run dolt ls
     [ "$status" -eq 0 ]
     ! [[ "$output" =~ "one_pk" ]] || false
-}
\ No newline at end of file
+}
+
+@test "sql-server: Create a temporary table and validate that it doesn't persist after a session closes" {
+    skiponwindows "Has dependencies that are missing on the Jenkins Windows installation."
+
+    cd repo1
+    start_sql_server repo1
+
+    # check no tables on master
+    server_query 1 "SHOW Tables" ""
+
+    # Create a temporary table with some indexes
+    server_query 1 "CREATE TEMPORARY TABLE one_pk (
```

Comment:
Need to verify that it still exists in this session in server mode, might need changes to the helper script for that

Reply:
Can this be moved to a separate pr? I'd love to take some time to scope out better testing for multiple queries in the same session. Disconnecting on every query isn't good enough.

Reply:
SGTM, must add a TODO in the meantime